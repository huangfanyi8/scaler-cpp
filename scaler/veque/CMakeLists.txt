cmake_minimum_required(VERSION 3.14)
project(veque)

set(CMAKE_CXX_STANDARD  23)

add_executable(test performance/main.cpp)