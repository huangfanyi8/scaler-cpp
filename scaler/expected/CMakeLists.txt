cmake_minimum_required(VERSION 3.20)
project(expected)

set(CMAKE_CXX_STANDARD 23)

add_library(json STATIC json.cpp)
target_include_directories(json PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
add_executable(expected main.cpp)
add_executable(test test_expected.cpp)
add_executable(json_test json_test.cpp)

